__tests__/api.test.js

```diff
@@ -0,0 +1,39 @@
+const request = require('supertest')
+const app = require('../app')
+const testData = require('../db/data/test-data/index')
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+
+beforeEach(() => {
+    return seed(testData)
+})
+
+afterAll(() => {
+    db.end()
+})
+
+describe('GET /api/topics testing', () => {
+    test('each object in the array should contain the properties: slug, description.', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((response) => {
+            console.log(response.body)
+            const topics = response.body.topics
+            expect(topics.length).toBe(3)
+            topics.forEach((topic) => {
+                expect(topic).toMatchObject(({slug: expect.any(String),
+                    description: expect.any(String)
+                }));
+            })
+        })
+    })
+    test('returns a 404 internal server error if the endpoint is not correct', () => {
```

Comment:
This description isn't accurate. It isn't an internal server error, it's not found

Also this test should be in it's own describe block as it will catch ALL bad urls, not just ones starting /api/topics